src/react/components/pages/predict/predictPage.tsx

```diff
@@ -684,6 +684,7 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
             });
         if (this.appInsights) {
             this.appInsights.trackEvent({ name: "ANALYZE_EVENT" });
+            this.appInsights.context.application.ver = constants.apiVersion;
```

Comment:
Can we set this when initialize appInsights?

Reply:
I'll check.

Reply:
done